src/foam/core/stdlib.js

```diff
@@ -433,7 +433,8 @@ foam.LIB({
       var argNames = foam.Function.argNames(fn);
       var args = [];
       for ( var i = 0 ; i < argNames.length ; i++ ) {
-        var a = source[argNames[i]];
+        var a = argNames[i].indexOf('$') == -1 ? source[argNames[i]] :
```

Comment:
From: https://github.com/foam-framework/foam2/pull/2511
These changes break travis by the look of it. I'll fix that in my PR. Undo these changes from your PR since they're kinda unrelated to what your PR is doing.

Reply:
Done.